### PR TITLE
Task history

### PR DESCRIPTION
**Pull request by MonCaptain**
Can now add/delete task from the frontend and have the changes be reflected on the backend.

### CODE REVIEW

File: react-app/pomodoro-timer-api/routes/task.js [hunk 1]
```diff
@@ -45,12 +61,26 @@ router.get("/pending", security.requireAuthenticatedUser, async (req, res, next)
 }
 });
 
+//  (post with name or put with no name) for updating task
+router.post("/update", async (req, res, next) => {
+  try {
+    console.log(res.locals.user)
```
Comment (Rebecca-CYLi) [inline react-app/pomodoro-timer-api/routes/task.js]: console log?

File: react-app/pomodoro-timer-api/routes/task.js [hunk 2]
```diff
@@ -45,12 +61,26 @@ router.get("/pending", security.requireAuthenticatedUser, async (req, res, next)
...
+  } catch (err) {
+    next(err);
+  }
+});
+
 // delete task
-router.delete("/::task", async function (req, res, next) {
-  console.log(req.params.task);
+router.delete("/:taskId", security.requireAuthenticatedUser, async function (req, res, next) {
+  console.log("deleting",req.params.taskId);
```
Comment (Rebecca-CYLi) [inline react-app/pomodoro-timer-api/routes/task.js]: console log?

File: react-app/pomodoro-timer-ui/src/Components/TodoTab/TodoForm.jsx [hunk 3]
```diff
@@ -24,6 +24,7 @@ export default function TodoForm() {
   function handleOnSubmitTask(event) {
     event.preventDefault();
     if (todo.task.trim()) {
+      console.log("adding todo...")
```
Comment (Rebecca-CYLi) [inline react-app/pomodoro-timer-ui/src/Components/TodoTab/TodoForm.jsx]: console log?

File: react-app/pomodoro-timer-ui/src/contexts/TodoHistoryContext.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  useEffect(() => {
+    async function getCompletedTasks(){
+        const {data, error} = await apiClient.getCompletedTasks();
+        if (data) setCompletedTodos(data)
+        if (error) setError(error)
+      }
+      getCompletedTasks();
+  }, []);
+
+  console.log("completedTodos", completedTodos)
```
Comment (Rebecca-CYLi) [inline react-app/pomodoro-timer-ui/src/contexts/TodoHistoryContext.jsx]: console log?

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
console logs

**Review by MonCaptain — COMMENTED** [review]
removed all console.logs. Thank you for always catching them.

**Review by Rebecca-CYLi — APPROVED** [review]
LGTM

**Comment by MonCaptain** [comment]
> LGTM

ty as always

**Pull request merged**